 'constexpr' needed for in-class initialization of non-int static data member

However, constexpr is c++11, so better no require it just for this. Just do the
initialization outside of the class.

diff --git codels/ukf_filter.cc codels/ukf_filter.cc
index ec1c0d3..a5d1a81 100644
--- codels/ukf_filter.cc
+++ codels/ukf_filter.cc
@@ -28,9 +28,9 @@ using namespace ukf;

 class filter_s::impl_s
 {
-  static const double alpha = 0.5;
-  static const double beta = 2.;
-  static const double kappa = 0.;
+  static const double alpha;
+  static const double beta;
+  static const double kappa;

   static inline double lambda() {
     return alpha * alpha * (state_s::dof + kappa) - state_s::dof;
@@ -171,6 +171,11 @@ class filter_s::impl_s
   }
 };

+/* --- filter_s ------------------------------------------------------------ */
+
+const double filter_s::impl_s::alpha = 0.5;
+const double filter_s::impl_s::beta = 2.;
+const double filter_s::impl_s::kappa = 0.;


 /* --- filter_s::filter_s -------------------------------------------------- */
